components/forms/AddToGroupsForm.js

```diff
@@ -16,21 +16,34 @@ export default function AddToGroupsForm({ groups }) {
             <h4 className="mb-4">Groups</h4>
             <h5>System supplied</h5>
             <div className="groups__container">
+                {capabilities.includes(promoteToSuperAdmin) && <div className="groups__item" >
```

Comment:
I'm not comfortable giving 'Super Admin' special treatment. 

I suggest we use the parameterized capabilities, so we have something like:

`promote_to(super_admin)`
`promote_to(admin)`
`promote_to(registered_user)`
etc.

Reply:
The `promoteToSuperAdmin` variable is somewhat similar to that. I just put it in a variable in the constants folder.

It is equivalent to this: `const promoteToSuperAdmin = `${assignToGroup}(${SUPER_ADMIN_ID})`;`

Where `assignToGroup` is the `assign_to_group` capability and the `SUPER_ADMIN_ID` is the id # of the super admins, therefore it is equal to `assign_to_group(1)`

Should I specifically change this to `capabilities.includes('promote_to(super_admin)')`, where `promote_to(super_admin)` is a string? (or maybe `capabilities.includes('assign_to_group(1)')`

Reply:
🤔